ui/pages/home/beta/beta-home-footer.component.js

```diff
@@ -13,12 +13,15 @@ const BetaHomeFooter = () => {
   const t = useI18nContext();
   const trackEvent = useContext(MetaMetricsContext);
 
+  const SUPPORT_LINK =
+    'https://docs.google.com/forms/d/e/1FAIpQLSfpkpeVMvfFw_1CWzrs5M3TEqE9VPMSspLPJgk1u4ZyUkQRbg/viewform?usp=sharing';
```

Comment:
FYI this form is currently private when viewed in a fresh browser session )only visible for ConsenSys users I'm guessing).

Just noting in case this was unintended. I'm guessing this was intentional, and that we're planning on making it public upon beta launch.